Sources/API/HTTPConstants.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  HTTPConstants.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public enum HTTPHeaderField {
+    
+    public static let acceptEncoding = "Accept-Encoding"
+    public static let acceptLanguage = "Accept-Language"
+    public static let userAgent = "User-Agent"
+    public static let etag = "ETag"
+    public static let ifNoneMatch = "If-None-Match"
+    public static let moreInfo = "X-DuckDuckGo-MoreInfo"
+    
+}
+
+public enum HTTPMethod: String {
+    
+    case get = "GET"
+    case head = "HEAD"
+    case post = "POST"
+    case put = "PUT"
+    case delete = "DELETE"
+    case connect = "CONNECT"
+    case options = "OPTIONS"
+    case trace = "TRACE"
+    case patch = "PATCH"
+    
+}
```

Comment:
Shouldn't these be put in a tighter scope instead of on Module level? E.g. under APIRequest